astropy/coordinates/tests/test_angles.py

```diff
@@ -572,6 +572,12 @@ def test_angle_string():
     a = Angle("10'")
     assert_allclose(a.value, 10.0)
     assert a.unit == u.arcminute
+    a = Angle('00:00:60N', u.deg)
+    assert str(a) == '0d01m00s'
+    a = Angle('00:00:10S', u.hour)
```

Comment:
Yikes, that is ugly and ideally would not work (cannot use hours for N/S). But I can see that that is not really part of the parser, so probably not easy to prevent...

Reply:
@mhvk I does work should I not allow the directions in the **colon** way of writing the angles?

Reply:
The problem is that the parser does not know what the units will be, degrees or hours, so I think there is no way to avoid it. And I do think we should allow it for the colons too.